### ISSUE TITLE
Item Lasers will randomly disconnect.

### ISSUE THREAD

**Issue by CactusCool** [Ellpeck/ActuallyAdditions#1379]
I along with a lot of people in the nomifactory discord have noticed that when automating assembly lines with item lasers they will randomly disconnect for no apparent reason.

**Comment by KleinGrrmpf** [comment]
Yeah, there have already been issues opened here and closed again because the mod author couldn't reproduce it. My guess is that it happens when there are either a lot of lasers in a network or a lot of items in a network. And it's not just upon relogging but also during a session. Not limited to Nomifactory either. I really wish this would get fixed...

**Comment by Redderpears** [comment]
This is in a nomifactory save, and I just wanted to comment to point out the interesting pattern in which mine disconnected.

This is how it looks when I first noticed it.
![image](https://github.com/Ellpeck/ActuallyAdditions/assets/92650935/28fee7f1-82e8-4d1a-b9c9-22807cb0b900)

This is how it looks after reconnecting all of the lasers
![image](https://github.com/Ellpeck/ActuallyAdditions/assets/92650935/44e3e6a9-a5d8-4f2b-810d-fcb3d0d04c35)

This is how it looks after placing one enderio energy conduit on a large (~1581 conduit) system, causing a large (1-2 second) lag spike
![image](https://github.com/Ellpeck/ActuallyAdditions/assets/92650935/0e0d2a0a-c806-4162-a98b-3edb4abcd178)
